Whatsapp_Chat_Exporter/__main__.py

```diff
@@ -545,11 +545,11 @@ def main():
             args.offline,
             args.size,
             args.no_avatar,
-            not args.filter_empty
+            args.disable_filter_empty
         )
 
     if args.json and not args.import_json:
-        if not args.filter_empty:
+        if args.disable_filter_empty:
```

Comment:
I think it should be `if not`
if you disable the filter do you don't want to run the next like that is actually filtering out the empty chats 
See also the comment on the args parser part

Reply:
As discussed in args parser part.